lib/va_notify/notification_email.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+# Library for VA Notify
+module VANotify
```

Comment:
@wayne-weibel with the new monitor classees should this maybe go under `logging/va_notify.rb` so its right there with `monitor` and `sidekiq`?

Reply:
50/50 - this monitoring is specific to this module.  can debate and move once the general monitoring changes are in too